packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss

```diff
@@ -134,38 +141,41 @@
       display: none;
     }
 
-    padding-left: units(
-      $theme-site-margins-mobile-width - $theme-alert-bar-width
-    );
-    @include at-media($theme-site-margins-breakpoint) {
-      padding-left: 2 * $alert-icon-optical-padding;
+    .usa-alert__heading,
+    .usa-alert__text {
+      margin-left: 0;
+
+      @include at-media($theme-site-margins-breakpoint) {
+        margin-left: 0;
+      }
```

Comment:
**Question:** Same as before, do we need this media query if we're applying the same margin as the line above?

Reply:
Good catch - removed the redundant/unnecessary media queries in [8e5295c](https://github.com/uswds/uswds/pull/5636/commits/8e5295cd51d7fdd232d96f514025210d6d987b5f)